tests/test_physics.py

```diff
@@ -1213,6 +1222,64 @@ def test_articulated_object_joint_motors(test_asset):
         if "amass_male" not in test_asset:
             assert np.allclose(new_vel_target, robot.joint_velocities, atol=0.06)
 
+            for link_id in robot.get_link_ids():
```

Comment:
I think this test can be significantly reduced.
You could query and validate the joint motor torques(line 1268 below) at line 1217 above. There, the simulator should be actively actuating all joint motors with a velocity target, so you can skip all this motor setup logic.